.github/workflows/generate_third_party_licenses.yml

```diff
@@ -3,6 +3,7 @@ name: Generate THIRD-PARTY-LICENSES
 on:
   push:
     tags: [ "v[0-9]+.[0-9]+.[0-9]+" ]
+    branches: [ "dependabot/*", "main", "workflow/*" ]
```

Comment:
Only Github team members can create a new branch directly on the repository, right?

Reply:
Yes, and dependabot. According to this help page, anyone with the 'write' permission can create branches: https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-a-branch-for-an-issue#creating-a-branch-for-an-issue

I've double checked our permissions, and only people in [s3-connector-for-pytorch](https://github.com/orgs/awslabs/teams/s3-connector-for-pytorch) have permissions to write.